src/libs/API.js

```diff
@@ -751,6 +751,40 @@ function BankAccount_Create(parameters) {
     return Network.post(commandName, parameters, CONST.NETWORK.METHOD.POST, true);
 }
 
+/**
+ * @param {Object} parameters
+ * @param {String[]} data
+ * @returns {Promise}
+ */
+function Mobile_GetConstants(parameters) {
+    const commandName = 'Mobile_GetConstants';
+    requireParameters(['data'], parameters, commandName);
+
+    // For some reason, the Mobile_GetConstants endpoint requires a JSON string, so we need to stringify the data param
```

Comment:
I think the reason has nothing to do with the endpoint we just can't send an object in `FormData`

Reply:
Ah yep, probably best to rewrite this comment to avoid confusion.

Reply:
Just a note, this function was removed in https://github.com/Expensify/Expensify.cash/pull/3016

Reply:
Thanks for the head up.

Reply:
So it looks like this PR overrode that removal, I'll just check with Tim to see if there was a reason for removing this (other than because it was unused)

Reply:
@Julesssss  I was looking to send an updated PR. Should I just send one with correcting fetchAllReports(true, true)?

Reply:
Hi @rameshhpathak, I think this comment can be updated, as Marc said. Other than that I believe it's just the ` fetchAllReports(true, true)` change. Thanks!

Reply:
I've resolved this as part of a PR that [touches the currency logic.](https://github.com/Expensify/Expensify.cash/pull/3514#issue-666950596)